yajsapi/executor/index.ts

```diff
@@ -419,90 +412,105 @@ export class Executor implements ComputationHistory {
       logger.debug("Stopped processing debit notes.");
     }
 
-    async function find_offers(): Promise<void> {
-      let _subscription: Subscription;
+    async function find_offers_for_subscription(subscription: Subscription): Promise<void> {
```

Comment:
Does this PR also contain changes not related to agreement pool? If so, it would be useful to list them in the PR description.

Reply:
Fixed (`master` branch was not up to date).